rbi/stdlib/json.rbi

```diff
@@ -378,6 +378,329 @@ class JSON::GenericObject < OpenStruct
   def self.load(source, proc=T.unsafe(nil), opts=T.unsafe(nil)); end
 end
 
+module JSON::Ext
+end
+
+module JSON::Ext::Generator
+end
+
+module JSON::Ext::Generator::GeneratorMethods
+end
+
+module JSON::Ext::Generator::GeneratorMethods::Array
+  sig do
+    params(
+      _: ::T.untyped
+    )
+    .returns(::String)
+  end
+  def to_json(*_); end
```

Comment:
Why did you define this as varargs, instead of just a single argument?
https://ruby-doc.org/stdlib-2.4.1/libdoc/json/rdoc/JSON/Ext/Generator/GeneratorMethods/Array.html

Reply:
The method definitions were created automatically, I just added the signatures. Documentation claims that there is a single parameter but, Ruby says otherwise:
```ruby
[2] pry(main)> Array.method(:to_json).parameters
=> [[:rest]]
```

Reply:
Ah, that makes sense. Thanks!